components/BaseComponents.js

```diff
@@ -0,0 +1,108 @@
+import styled from 'styled-components/native';
+
+export const TextButton = styled.Text`
+  font-family: poppins-regular;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #008550;
+`;
+
+export const TextButtonPressed = styled(TextButton)`
+  color: #004e14;
+`;
+
+export const TextButtonDisabled = styled(TextButton)`
+  color: #bdbdbd;
+`;
+
+export const ButtonContainer = styled.TouchableOpacity``;
+
+export const ButtonLabel = styled(TextButton)`
+  top: 25%;
+  bottom: 25%;
+  font-family: poppins-regular;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 14px;
+  line-height: 21px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.01;
+  text-transform: uppercase;
+  color: ${props => (props.filledButton ? '#ffffff' : '#000000')};
```

Comment:
Add something like this to every text style (or at least the ones with frequently varying colors like Body Caption and Title) but directly passing in the color name (once color names are defined) into the styled component with a default color if nothing is specified.
`color: ${props => (props.textColor ? props.textColor : 'text-primary')}`

cc @FranCogMents

Reply:
Thanks!!! Realized I was thinking about nested ternary operations the wrong way :~)